### ISSUE TITLE
数据集处理部分也许有一个小bug？

### ISSUE THREAD

**Issue by Lizhe1228** [slrl123/BSAFusion#16]
很好的一个工作，最近正在follow。复现时发现数据集处理部分也许有一个小bug？或是我没找到正确的方式。 设置的随机种子也许无法控制transform中的RandomChoice选择同一个方向？会出现img1和img2旋转方向不一致的问题。源代码部分如下：
       if self.mode == 'train':
            # make sure transform in same paradigm
            seed = torch.seed()
            torch.manual_seed(seed)
            model_img1 = self.transform(model_img1)
            torch.manual_seed(seed)
            model_img2 = self.transform(model_img2)
我自己换了一种方式
        if self.mode == 'train':
            concat = torch.cat([model_img1, model_img2], dim=0)
            concat = self.transform(concat)
            C1 = model_img1.shape[0]
            model_img1 = concat[:C1]
            model_img2 = concat[C1:]

**Comment by slrl123** [comment]
感谢您的关注，把图片对先cat，再旋转，然后再分开应该可以保证旋转方向的一致性。

**Comment by Lizhe1228** [comment]
您好，请问能提供一下testdata中PET和SPECT这两个模态图像未形变之前的RGB图像  现在代码中只用未形变的Y通道和形变后的RGB
